.github/workflows/build-binaries.yml

```diff
@@ -0,0 +1,72 @@
+name: Build Binaries
+on:
+  push:
+    branches:
+      - main
+      - "releases/*"
+
+jobs:
+  # Compile the binaries and upload artifacts
+  compile-binaries:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - os: ubuntu-latest
+            package-suffix: linux-amd64
+          - os: macos-latest
+            package-suffix: macos-amd64
+          - os: windows-latest
+            package-suffix: windows-amd64
+          - os: ubuntu-arm
+            package-suffix: linux-aarch64
+            # Need the 8 CPU version that has 12GB of RAM, the 4 CPU version
+            # only has 6 GB.
+            runsOn: buildjet-8vcpu-ubuntu-2204-arm
+    runs-on: ${{ matrix.runsOn || matrix.os }}
```

Comment:
Why do you need `runsOn` and `os`? Could just use `runsOn`

Reply:
Because having `os` consistent in the matrix makes the job list look pretty in the UI :-)